src/public/appInitialization.ts

```diff
@@ -3,41 +3,70 @@ import { version } from '../internal/constants';
 import { sendMessageToParent } from '../internal/communication';
 
 export namespace appInitialization {
+  export const notifyMessages = {
+    appLoaded: 'appInitialization.appLoaded',
+    success: 'appInitialization.success',
+    failure: 'appInitialization.failure',
```

Comment:
[NIT] this is something we should do for all sdk APIs. Probably rename it to export SDKAPIs = {
appInitialization: {
   appLoaded: 'appInitialization.appLoaded',
    success: 'appInitialization.success',
    failure: 'appInitialization.failure',
 initializedWithErrors: 'appInitialization.initializedWithErrors',
}

}
}

Reply:
Since this is exported under the appInitialization namespace, it is already available as a constant when accessing the package.  You can reference it like this:
I've noticed that none of the other message strings are exported. This is something we should fix in a separate PR. All the message string constants should be exposed so we can reference them directly. We can create a new object for SDKMessages.